src/axisym_foeppl_von_karman/axisym_fvk_elements.cc

```diff
@@ -525,66 +517,6 @@ namespace oomph
       norm += exact_soln[0] * exact_soln[0] * W;
       error += (exact_soln[0] - w_fe) * (exact_soln[0] - w_fe) * W;
     }
-
-    {
```

Comment:
Mention in commit that duplicate code was removed.

Reply:
Now mentioned in the commit which made this change